src/features/pageObjects/components/PageObjMenu.tsx

```diff
@@ -27,49 +26,39 @@ import { OnbrdTooltip } from "../../onboarding/components/OnbrdTooltip";
 import { OnboardingContext } from "../../onboarding/OnboardingProvider";
 
 interface Props {
-  id: PageObjectId;
-  name: string;
-  url: string;
-  locators?: ElementId[];
+  pageObject: PageObject;
   elements: Locator[];
-  library: ElementLibrary;
 }
 
-export const PageObjMenu: React.FC<Props> = ({ id, name, url, locators, elements, library }) => {
```

Comment:
why you refused object destructuring here? and in this PR at all?

Reply:
Because in many places code starts looking messy with passing more than 5-6 parameters and since `PageObject` is not that big object, I believe we can neglect not used properties for sake of readability.
But okay, if you think it’s better I’ll return destructuring.